src/hooks/useAsignFruit.js

```diff
@@ -0,0 +1,22 @@
+import {useState} from 'react'
+function getRandomInt(max) {
+    return Math.floor(Math.random() * max);
+  }
+export const useAsignFruit = () => {
+    const dictionary = new Map(); 
+    const [asignFruit, setState] = useState()
+    
+    
+    const createMap = (fruitList, personList) => {
+        
+        
+        dictionary.set(fruitList[0],personList[getRandomInt(2)] );
```

Comment:
Hello @HanierPena I wonder Why did you use the number 2 instead the length of the array ?